packages/api/src/controllers/helpers.ts

```diff
@@ -21,7 +21,7 @@ import { v4 as uuid } from "uuid";
 
 const ITERATIONS = 10000;
 const PAYMENT_FAILED_TIMEFRAME = 3 * 24 * 60 * 60 * 1000;
-const PULL_START_TIMEOUT = 60 * 1000;
```

Comment:
FYI just in case the problem with these were the `nginx` timeouts, I've just [increased those from 60s to 300s](https://github.com/livepeer/livepeer-infra/pull/2298) for the API, so the request wont' get cut anymore.